### ISSUE TITLE
[Bug] Ceiling in some biomes have wrong blocks

### ISSUE THREAD

**Issue by paulevsGitch** [quiqueck/BetterNether#14]
### What happened?

Some biomes have wrong ceiling blocks (found this in grasslands). In this case it is covered with Nether Moss instead of Netherrack

![2022-06-16_19 50 43](https://user-images.githubusercontent.com/17707380/174124518-157d5646-b845-45ff-8421-a0fa88bd45c7.png)

![2022-06-16_19 50 28](https://user-images.githubusercontent.com/17707380/174124517-73bc6e77-89e3-4792-bfef-675a0eeb6716.png)

### BetterNether

7.0.2

### BCLib

2.0.5

### Fabric API

0.56.0

### Fabric Loader

0.14.8

### Minecraft

1.19

### Relevant log output

```shell
none
```

### Other Mods

```shell
better-end-2.0.2
iris-mc1.19-1.2.5
sodium-fabric-mc1.19-0.4.2+build.16
```

**Comment by paulevsGitch** [comment]
Floor too, and probably filler blocks
![2022-06-16_19 54 31](https://user-images.githubusercontent.com/17707380/174125081-63905319-d043-4e46-bf7c-96d519eb5e4d.png)

**Comment by quiqueck** [comment]
Should be fixed in the just release 7.0.3 update.
